workspace/notebook/appeal_s78_curated_mipins.json

```diff
@@ -56,9 +56,9 @@
 				"source": [
 					"from notebookutils import mssparkutils\n",
 					"from datetime import datetime, date\n",
-					"from pyspark.sql.functions import from_utc_timestamp"
+					"from pyspark.sql.functions import from_utc_timestamp, col as col_name"
```

Comment:
It might be better to do `import pyspark.sql.functions as F` , then do `F.col` or `F.from_utc_timestamp`